src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier(random);
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100 + 1));
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)